var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -88,3 +88,9 @@ def cmake_args(self):
         ]
 
         return args
+
+    @run_after("install")
+    def darwin_fix(self):
+        # The shared library is not installed correctly on Darwin; fix this
+        if self.spec.satisfies("platform=darwin"):
```

Comment:
If this really applies to the shared library, you may want an extra check:

```suggestion
        if self.spec.satisfies("platform=darwin") and ("+shared" in self.spec):
```

Reply:
Good point, thanks!